src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -0,0 +1,181 @@
+        try {
+            const kbaDetails = (await API.get(`/rs/search?q=id:${this.props.rule.node_id }`)).data.response.docs[0];
+            this.setState({ kbaDetails });
+        } catch (error) {
+            this.props.addNotification({
+                variant: 'danger',
+                dismissable: true,
+                title: '',
+                description: 'KBA fetch failed.'
+            });
+        }
+    }
+
+    getSelectedItems = () => {
+        if (!this.props.entities || !this.props.entities.loaded) {
+            return [];
+        }
+
+        return this.props.entities.rows.filter(entity => entity.selected).map(entity => entity.id);
+    }
+
+    remediationDataProvider = () => {
+        return {
+            issues: [{
+                id: `advisor:${this.props.rule.rule_id}`,
+                description: this.props.rule.description
+            }],
+            systems: this.getSelectedItems()
+        };
+    }
+
+    ruleResolutionRisk = (rule) => {
+        return (rule.resolution_set.find(resolution => resolution.system_type === AppConstants.SYSTEM_TYPES.rhel).resolution_risk.risk);
+    };
+
+    ruleRebootRequired = (rule) => {
+        return rule.resolution_set.find(resolution => resolution.system_type === AppConstants.SYSTEM_TYPES.rhel).resolution_risk.reboot_required;
+    };
+
+    render () {
+        const { children, className, ruleFetchStatus, rule } = this.props;
+        const { kbaDetails } = this.state;
+        return (
+            <>
+                { ruleFetchStatus === 'pending' && (<Loading/>) }
+                { ruleFetchStatus === 'fulfilled' && (
+                    <Grid gutter='md' className={ className }>
+                        <GridItem md={ 8 } sm={ 12 }>
+                            <Grid>
+                                <GridItem className='pf-u-pb-md'>
+                                    {
+                                        typeof rule.summary === 'string' &&
+                                        Boolean(rule.summary) &&
+                                        <ReactMarkdown source={ rule.summary } escapeHtml={ false }/>
+                                    }
+                                </GridItem>
+                                { kbaDetails.view_uri && (
+                                    <GridItem className='pf-u-pb-md'>
+                                        <a href={ kbaDetails.view_uri }>
+                                            Knowledgebase Article
+                                        </a>
+                                    </GridItem>
+                                ) }
+                                { rule.tags && (
+                                    <GridItem>
+                                        Find Other Rules Related To:<br />
+                                        { rule.tags.split(' ').map((tagName, key) => (
+                                            <span className='pf-u-pr-md' key={ key }>
```

Comment:
Making each tag a span, each with its own `pf-u-pr-md` does not appear to jive with the mocks for this work (and is likely to get a lil wild when a rule has many tags), would recommend making it a string instead
<img width="1361" alt="screen shot 2019-03-06 at 12 59 55 pm" src="https://user-images.githubusercontent.com/6640236/53902793-f9bb4e80-400f-11e9-881f-17af2f912692.png">

<img width="1141" alt="screen shot 2019-03-06 at 1 00 08 pm" src="https://user-images.githubusercontent.com/6640236/53902792-f9bb4e80-400f-11e9-9d3b-94c5aaa25feb.png">